Cargo.toml

```diff
@@ -16,7 +16,7 @@ path = "lib.rs"
 [dependencies]
 slog = "2.4"
 lazy_static = "1.2"
-arc-swap = "0.4"
+arc-swap = "1.0"
```

Comment:
@reyk how about updating to "1.1" to address [RUSTSEC-2020-0091](https://rustsec.org/advisories/RUSTSEC-2020-0091) as well

```suggestion
arc-swap = "1.1"
```

Reply:
Good point, this PR predates the RUSTSEC advisory but never got merged. I can update it.